### PR TITLE
[TW-398] Update comments for ForagePANEditText and ForagePINEditText

### PR DESCRIPTION
**Pull request by kimberleehowley**
## What

This PR adds a comment to describe each `ForagePANEditText` and `ForagePINEditText`, and links to the relevant methods. 

## Why

Improving the docs -> better developer experience. 

## Test Plan

- ❌ | No unit tests for docs changes. 
- ✅ | I previewed the change locally.  
- ✅ |  The reviewer should manually test the changes in this PR. 

```
./gradlew dokkaHtml
npx http-serve reference-docs
```

### CODE REVIEW

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt [hunk 1]
```diff
@@ -20,6 +20,13 @@ import com.joinforage.forage.android.core.telemetry.Log
 import com.launchdarkly.sdk.android.LDConfig
 import com.verygoodsecurity.vgscollect.widget.VGSEditText
 
+/**
+ * A [ForageElement] that collects a customer's card number. You need a [ForagePINEditText] to call
+ * the methods to [check a card's balance][com.joinforage.forage.android.ForageSDK.checkBalance],
```
Comment (djoksimo) [inline forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt:25]:
```suggestion
 * A [ForageElement] that securely collects a customer's PIN. You need a [ForagePINEditText] to call
 * the methods to [check a card's balance][com.joinforage.forage.android.ForageSDK.checkBalance],
```

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt [hunk 2]
```diff
@@ -20,6 +20,13 @@ import com.joinforage.forage.android.core.telemetry.Log
...
+ * [collect a card PIN to defer payment capture to the server][com.joinforage.forage.android.ForageSDK.deferPaymentCapture],
+ * [capture a payment immediately][com.joinforage.forage.android.ForageSDK.capturePayment], or
+ * [refund a Payment][com.joinforage.forage.android.pos.ForageTerminalSDK.refundPayment] (POS only).
```
Comment (djoksimo) [inline forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt:28]: Thoughts on using the `* *` bullet point notation here?

**Review by djoksimo — CHANGES_REQUESTED** [review]
Great use of links and references to the other methods!

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt (hunk 1 above)
Comment (kimberleehowley) [reply forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt:25]: Sorry for the sloppy copy/pasta 🤦

File: forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt (hunk 2 above)
Comment (kimberleehowley) [reply forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt:28]: Me da igual! I'll push up that change shortly.

**Review by djoksimo — APPROVED** [review]

**Pull request merged**